tools/mdbuild.py

```diff
@@ -143,7 +143,7 @@ def post_process(self):
             ('%%CONTENT%%', self.parsed),
             ('%%DOCNAV%%', doc_nav),
             ('src="media/', 'src="../media/'),
-            ('src="./media/', 'src="../media/'),
+            ('src="./media/', 'src="./media/'),
```

Comment:
:laughing: now you've changed the other side of this! Leave poor mdbuild.py alone!